apps/vanilla/src/pages/index.css

```
@@ -1,5 +1,208 @@
 @import url("../global/index.css");
 
-.title {
-  color: var(--heading-color-1);
+/* Desktop */
+.container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  gap: var(--spacing-lg);
+}
+
+.content-wrapper {
+  display: flex;
+  gap: var(--spacing-lg);
+}
+
+/* Book related */
+.book-display {
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-xxl);
+  flex: 1;
+}
+
+.book-group {
+  display: flex;
+  gap: var(--spacing-sm);
+  flex-direction: column;
+}
+
+.book-group__heading {
+  position: sticky;
+  background: var(--surface-color);
+  padding: var(--spacing-md) 0;
+  top: 0;
+}
+
+.book-group__list {
+  gap: var(--spacing-md);
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.book-card {
+  width: 180px;
+  overflow: hidden;
+}
+
+.book-card__cover {
+  max-width: 100%;
+  aspect-ratio: 2/3;
+  border-radius: var(--border-radius-lg);
+  border: 1px solid var(--border-color);
+  overflow: hidden;
+}
+
+.book-card__cover img {
+  object-fit: fill;
```

`fill` is distorting/stretching image, let's use `cover`.

```suggestion
  object-fit: cover;
```

---

Thanks! The problem has been resolved